src/status_im/chat/models.cljs

```diff
@@ -259,17 +259,23 @@
               (transport.filters/load-chat chat-id)
               (navigate-to-chat chat-id))))
 
+(defn valid-pubchat-topic? [topic]
+  (re-matches #"^[0-9a-z-]*$" topic))
```

Comment:
we have this in `status-im.ui.screens.add-new.new-public-chat.db`

Reply:
thanks, fixed